apisix/plugins/basic-auth.lua

```diff
@@ -30,7 +30,12 @@ local consumers_lrucache = core.lrucache.new({
 local schema = {
     type = "object",
     title = "work with route or service object",
-    properties = {},
+    properties = {
+        hide_auth_header = {
```

Comment:
`hide_credentials` would be better? Kong uses this field in their basic-auth.

Reply:
Done.